### PR TITLE
Improve time input

### PR DESCRIPTION
**Pull request by thecalcc**

### CODE REVIEW

File: app/styles/components/_theme-selector.scss [hunk 1]
```diff
@@ -145,6 +145,10 @@
 }
 
 [data-theme="dark-ui"] {
+    input[type="time"]::-webkit-calendar-picker-indicator {
```
Comment (tomaskikutis) [inline app/styles/components/_theme-selector.scss:148]: webkit is chromium, what about firefox? does it work there outside the box?

File: app/styles/form-elements/_inputs.scss [hunk 2]
```diff
@@ -654,6 +654,7 @@
         grid-column: 2/4;
     }
     .sd-input__input-container {
+        min-width: 110px;
```
Comment (tomaskikutis) [inline app/styles/form-elements/_inputs.scss]: is this for all inputs? Shouldn't it only be for time picker?

File: app/styles/components/_theme-selector.scss (hunk 1 above)
Comment (thecalcc) [reply app/styles/components/_theme-selector.scss:148]: Firefox's native time input doesn't contain an icon like Chrome does so we don't need to think about that.

File: app/styles/form-elements/_inputs.scss (hunk 2 above)
Comment (thecalcc) [reply app/styles/form-elements/_inputs.scss]: Yes, changed it so that we apply it to this class only if it contains a time input

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**